clients/git/client.go

```diff
@@ -201,9 +199,81 @@ func (c *Client) GetFileContent(filename string) ([]byte, error) {
 	return nil, nil
 }
 
+func (c *Client) IsArchived() (bool, error) {
+	return false, nil
+}
+
+func (c *Client) URI() string {
+	return ""
```

Comment:
should this surface the underlying `repo.URI`?

Reply:
👍